### PR TITLE
Fonts overhaul

### PR DESCRIPTION
**Pull request by marcin-serwin**
This PR extends mainly extends font-specific sections about `fontspec`, font changing and colors.

### CODE REVIEW

**Comment by oetiker** [comment]
@marcin-serwin cool, will be reviewing shortly ... 

I have been thinking about changing the name of the document from LaTeXe to LaTeX ... since LaTeXe is not really a thing anymore ... probably more confusing to people these days ... 

what do you think ?

**Comment by marcin-serwin** [comment]
> I have been thinking about changing the name of the document from LaTeXe to LaTeX ... since LaTeXe is not really a thing anymore ... probably more confusing to people these days ...

Believe it or not, this was supposed to be my next PR :>

On the topic on meta changes, I was also thinking about updating licensing a bit: bump the GPL to version 3 and release the code examples under CC0. The GPL3 version isn't that much different, but contains updated wording that works better in international context. The code examples are too small to be copyrightable anyway, but it's nice to make this explicit so people don't have to worry. This is also the way taken by similar projects, e.g. [Learn LaTeX](https://www.learnlatex.org/en/licenses).

What do you think?

**Comment by oetiker** [comment]
Hi @marcin-serwin,  since gpl2 allows to use any later gpl as well ... yea, why not ... lets go gpl3.

File: book/src/spec.tex [hunk 1]
```diff
@@ -617,385 +596,272 @@ \section{Installing Extra Packages}\label{sec:Packages}
...
-\end{code}
-
-When creating PDFs destined for printing, coloured links are not a
-good thing as they end up in gray in the final output, making it
-difficult to read. Use colour frames, which are not printed:
-\begin{code}
-\begin{minted}{latex}
+\begingroup
+\LshortExampleSetup{hide_hyperref=false}
+Initial release of \TeX{} predated the PDF format by nearly 16 years. The
```
Comment (oetiker) [inline book/src/spec.tex:601]:
```suggestion
The initial release of \TeX{} predated the PDF format by nearly 16 years. The
```

File: book/src/spec.tex [hunk 2]
```diff
@@ -617,385 +596,272 @@ \section{Installing Extra Packages}\label{sec:Packages}
...
+original output files it produced---\eei{.dvi}s---were not meant to be read
+read on screen, only printed. Today many produced documents are never or seldom
```
Comment (oetiker) [inline book/src/spec.tex:603]:
```suggestion
on screen, only printed. Today many documents are never or seldom
```

File: book/src/spec.tex [hunk 3]
```diff
@@ -617,385 +596,272 @@ \section{Installing Extra Packages}\label{sec:Packages}
...
+The reference to this section
+now looks like that:\footnote{
+  Footnotes are also
+  made into hyperlinks.}
+Section~\ref{hyperlinks} is
+on page~\pageref{hyperlinks}.
+The hyperref bibliographic
+entry is~\cite{pack:hyperref}.
+\end{example}
+As you can see, by default links are marked by a red box around them. This box
```
Comment (oetiker) [inline book/src/spec.tex:642]:
```suggestion
By default links are marked by a red box around them. This box
```

File: book/src/spec.tex [hunk 4]
```diff
@@ -617,385 +596,272 @@ \section{Installing Extra Packages}\label{sec:Packages}
...
+is only visible when viewing the document on screen and will not be printed.
+They are, however, rather ugly, so you may want to add the \cargv{colorlinks}
```
Comment (oetiker) [inline book/src/spec.tex:644]:
```suggestion
The boxes are, however, rather ugly, so you may want to add the \cargv{colorlinks}
```

**Review by oetiker — APPROVED** [review]

**Comment by marcin-serwin** [comment]
I've rebased it onto #45 so it should be merged after it.

**Pull request merged**